src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("deadline")) {
+                    Task newTask = Chatter.taskBank.addDeadline(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("event")) {
+                    Task newTask = Chatter.taskBank.addEvent(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("delete")) {
+                    Chatter.deleteTask(sentence);
+                } else if (sentence.startsWith("clear")) {
+                    Chatter.clear();
+                } else {
+                    throw new IrregularInputException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(" +
+                            "\nPlease try again!");
+                }
+            } catch (IrregularInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (RepeatedCompletionException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (EmptyInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (NumberFormatException e) {
+                printDashLine();
+                System.out.printf("Not a valid number. Try again%n");
+                printDashLine();
+            }
+        }
+    }
+
+    static void add(Task newTask) {
+        printDashLine();
+        System.out.printf("Got it. I've added this task: %n " +
+                newTask +
+                "%nNow you have " + taskBank.getTaskSize() +
+                " tasks in the list.%n");
+        printDashLine();
+        taskBank.exportTasks();
+    }
+
+    static void bye() {
+        printDashLine();
+        System.out.printf("Bye. Hope to see you again soon!%n");
+        printDashLine();
+    }
+
+    static void list() {
+        printDashLine();
+        System.out.printf("Here are the tasks in your list:%n");
+        taskBank.printList();
+        printDashLine();
+    }
+
+    static void done(String sentence) throws RepeatedCompletionException, NumberFormatException {
+        String[] words = sentence.split(" ");
+        int targetIndex = Integer.parseInt(words[1]) - 1;
+        Task targetTask = taskBank.searchTask(targetIndex);
+        targetTask.markAsDone();
+        taskBank.exportTasks();
+        System.out.printf("Nice! I've marked this task as done: %n");
+        System.out.println("  " + targetTask);
+        printDashLine();
+    }
+
+    static void deleteTask(String sentence) {
+        String[] words = sentence.split(" ");
+        int targetIndex = Integer.parseInt(words[1]) - 1;
```

Comment:
Magic literals should be avoided by defining them as a constant.

Reply:
Hi, may I know which is the magic literal that you are referring to?